synthtool/gcp/common.py

```diff
@@ -58,6 +59,11 @@ def _generic_library(self, directory: str, **kwargs) -> Path:
 
         t = templates.TemplateGroup(self._template_root / directory, self.excludes)
 
+        repo = kwargs["metadata"]["repository"]
+        github_req = requests.get(f"https://api.github.com/repos/{repo}")
```

Comment:
@SurferJeffAtGoogle Will this be problematic for owl-bot (as it's not hermetic)?

Reply:
I think Jeff is out til Friday? Is there anyone else we can ask about this? Or do you have thoughts on this?

Reply:
To me this seems like it's probably okay, as it will be applied in the post-processor, and will stay consistent for the repo itself?